app/assets/javascripts/base.js.erb

```diff
@@ -132,6 +132,7 @@ var instantClick
           skipLink.focus();
         }
       }
+      document.getElementById('page-route-change').textContent = `Navigated to ${document.title}`;
```

Comment:
```suggestion
      document.getElementById('page-route-change').textContent = title;
```
nitpick (blocking) - We should use the `title` variable that's already been initialized in this function, as sometimes the document title hasn't changed yet (see lines 164 onwards). Also if we remove 'navigated to...', it's more similar to default screen reader behaviour, so hopefully should be a better UX for users